src/polydb.jl

```diff
@@ -0,0 +1,301 @@
+module Polydb
+
+import Polymake: call_function
+
+using Polymake
+
+# julia itself also has a cert.pem but this one should be more recent
+# and provides a variable for the path
+using MozillaCACerts_jll
+
+using Mongoc
+
+#Polymake.Polydb's types store information via
+# a corresponding Mongoc type variable
+struct Database
+   mdb::Mongoc.Database
+end
+
+struct Collection{T}
+   mcol::Mongoc.Collection
+end
+
+struct Cursor{T}
+   mcursor::Mongoc.Cursor{Mongoc.Collection}
+end
+
+# connects to the Polydb and
+# returns a Polymake.Polydb.Database instance
+function get_db()
+   # we explicitly set the cacert file, otherwise we might get connection errors because the certificate cannot be validated
+   client = Mongoc.Client("mongodb://polymake:database@db.polymake.org/?authSource=admin&ssl=true&sslCertificateAuthorityFile=$(cacert)")
+   return Database(client["polydb"])
+end
+
+# returns a Polymake.Polydb.Collection instance with the given name
+# sections and collections in the name are connected with the '.' sign,
+# i.e. names = "Polytopes.Lattice.SmoothReflexive"
+function get_collection(db::Database, name::String)
```

Comment:
since the Database is so close to `Mongoc.Database` why not make it into
```julia
Base.getindex(db::Database, name::AbstractString) = Collection{Polymake.BigObject}(db.mdb[name])
```
also are we sure these will be always `BigObjects`?

this should actually dispatch through the next method as 
`Base.getindex(db::Database, name::AbstractString) = db[name, Polymake.BigObject]`

Reply:
The polymake database contains only BigObjects (that might change at some point but probably not so soon).